src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -54,8 +59,35 @@ function updateDataDump() {
             '' : dataPushToTextarea + userDataRows); // only pushes header string if userDataRows is not empty
 }
 
+function getAjaxStudentList() {
+    const $spreadsheetForm = $('#student-data-spreadsheet-form');
+
+    $.ajax({
+        type: 'POST',
+        url: '/page/instructorCourseEnrollAjaxPage',
+        cache: false,
+        data: {
+            courseid: $spreadsheetForm.children(`input[name="${ParamsNames.COURSE_ID}"]`).val(),
+            user: $spreadsheetForm.children(`input[name="${ParamsNames.USER_ID}"]`).val(),
+        },
+        success(data) {
+            const studentsData = data.students;
+            const studentsDataList = [];
+            studentsData.forEach((student) => {
+                let tempStudentsData = [];
+                handsontable.getColHeader().forEach((header) => {
+                    tempStudentsData.push(student[unCapitalizeFirstLetter(header)]);
+                });
+                studentsDataList.push(tempStudentsData);
+            });
+            handsontable.loadData(studentsDataList);
+        },
```

Comment:
How about failure? Should notify the user as well

Reply:
Failure is taken care of in the newer commits :)